app/services/process_feed_service.rb

```diff
@@ -50,6 +49,7 @@ def create_status
       if verb == :share
         original_status, = status_from_xml(@xml.at_xpath('.//activity:object', activity: TagManager::AS_XMLNS))
         status.reblog    = original_status
+        status.text      = ''
```

Comment:
Not necessary for the fix, but I noticed that remote reblogs had their wrapper text stored, which is inconsistent with local reblogs (empty text). That storage space is wasted. Also, when a bug like #1114 happens, in order of best-caseness:

1. do not have that bug happen at all
2. display "RT [original author] [text]" instead of just "[text]"
3. display an empty toot to make it obvious it's broken and not misattribute content visually